admin/acceptance_tests/themes_ordering.py

```diff
@@ -0,0 +1,56 @@
+# pylint: disable=no-self-use
+
+import pytest
+
+from .treegroup_tests import TestTreeGroup
+
+
+@pytest.fixture(scope='class')
+@pytest.mark.usefixtures('dbsession')
+def themes_ordering_test_data(dbsession):
+    from c2cgeoportal_commons.models.main import Theme
+
+    dbsession.begin_nested()
+
+    themes = []
+    for i in range(0, 25):
+        theme = Theme(name='theme_{}'.format(i),
+                      ordering=100)
+        dbsession.add(theme)
+        themes.append(theme)
+
+    dbsession.flush()
+    yield {
+        'themes': themes
+    }
+
+    dbsession.rollback()
+
+
+@pytest.mark.usefixtures('themes_ordering_test_data', 'transact', 'test_app')
+class TestThemesOrdering(TestTreeGroup):
+
+    _prefix = '/layertree/ordering'
+
+    def test_edit(self, test_app, themes_ordering_test_data, dbsession):
+        resp = self.get(test_app, status=200)
+        form = resp.form
+
+        self.check_children(
+            form,
+            'themes',
+            [{
+                'label': theme.name,
+                'values': {
+                    'id': str(theme.id)
+                }
+            } for theme in sorted(themes_ordering_test_data['themes'],
+                                  key=lambda t: (t.ordering, t.name))]
```

Comment:
one should use same variable as later, i.e. theme, in order to focus on test spirit ?